cassandra/src/test/resources/cassandra.yaml

```diff
@@ -0,0 +1,586 @@
+# Cassandra storage config YAML
```

Comment:
ps @michaelsembwever I'm thinking these test artifacts, if we choose to publish them, would make more sense as a test jar in the -core (java project) vs the scala one. That allows impls to drag in less conflicting deps to use them.